src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -30,8 +32,14 @@ public static double runMean(){
         return 0;
     }
 
-    public static double runVariance(){
-        return 0;
+    public static void runVariance(){
+        System.out.println("Run task Rainfall");
```

Comment:
Changes are needed:
or specify that it's variance part of task
or mean and variance should be merged in one run

Reply:
I meant to do it, but I forgot
done